beetmoverscript/task.py

```diff
@@ -22,26 +21,38 @@ def validate_task_schema(context):
     scriptworker.client.validate_json_schema(context.task, task_schema)
 
 
-def validate_task_scopes(context, manifest):
-    # make sure scopes exist and are properly set
-    scopes = context.task['scopes']
-    for scope in scopes:
-        if scope.startswith("project:releng:beetmover:"):
-            signing_cert_name = scope.split(':')[-1]
-            if re.search('^[0-9A-Za-z_-]+$', signing_cert_name) is not None:
-                break
-            log.warning('scope {} is malformed, skipping!'.format(scope))
-    else:
-        log.critical("No beetmover scopes!")
-        sys.exit(3)
-
-    # prevent scopes from low-security tree write in a release-type bucket
-    for k, v in RESTRICTED_BUCKET_PATHS.items():
-        for path in (manifest['s3_prefix_dated'], manifest['s3_prefix_latest']):
-            if path.startswith(k) and signing_cert_name != v:
-                log.critical("Munged low-security tree scopes trying to access"
-                             " nightly/release buckets for beetmover")
-                sys.exit(3)
+def get_task_cert_type(task):
```

Comment:
Ok, I have a few problems with this function.

* there's a lot of cert and signing verbiage here; I'm guessing this is because you copied the signingscript function?  We probably want more beetmover-type verbiage
* we split the credentials/bucket info from the behavior/action so we could control them separately.  That's so the behavior/action can be run on try, as long as we're using the staging bucket.  So the behavior/action scope isn't restricted; the credentials/bucket scope is.  Where do you check that scope?

In a signingscript analogy, the behavior/action is comparable to signing formats, though we're going to restrict the task to one.  We don't care if you're asking to sign with gpg or jar signing, as long as you have the scope to request it.  However, we want to restrict the ability to ask for signing with the nightly or release keys. These map to the nightly and release buckets / credentials for beetmover.